src/test/java/com/datasonnet/XMLWriterTest.java

```diff
@@ -15,16 +16,20 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
 
 public class XMLWriterTest {
-    
+
+    @BeforeAll
+    static void registerPlugins() throws Exception {
+        DataFormatService.getInstance().findAndRegisterPlugins();
```

Comment:
I think we've got this a few places right now so merging may create some conflicts, but they're easy ones so whatever